### ISSUE TITLE
allow specifying directory to save into

### ISSUE THREAD

**Issue by gnowxilef** [dreamer/scrot#4]

**Comment by mro** [comment]
try `-e` or e.g.

    $  scrot "$HOME/Pictures/shot-%Y-%m-%dT%H%M%S.png"

**Comment by gnowxilef** [comment]
sorry, i mean to say allow just specifying the directory and let scrot still handle the filename.

**Comment by mro** [comment]
http://lmgtfy.com/?q=scrot+directory

**Comment by gnowxilef** [comment]
sure, but that is running an external command. would be great if scrot supported that natively.

**Comment by mro** [comment]
no, IMO it wouldn't. It's perfectly fine however for one cmdline tool to leverage another. https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well
